populus/compilation/backends/solc_auto.py

```diff
@@ -16,37 +16,30 @@
 from .base import (
     BaseCompilerBackend,
 )
-from .solc_combined_json import (
-    SolcCombinedJSONBackend,
-)
+
 from .solc_standard_json import (
     SolcStandardJSONBackend,
 )
 
 
-def get_solc_backend_class_for_version(solc_version):
-    if is_string(solc_version):
+def get_solc_backend_class_for_version(solc_version=None,settings=None):
+
+    if solc_version == None:
+        solc_version = get_solc_version()
+
+    elif is_string(solc_version):
         solc_version = Version(solc_version)
 
-    if solc_version in Spec('<=0.4.8'):
-        return SolcCombinedJSONBackend
-    elif solc_version in Spec('>=0.4.11'):
-        return SolcStandardJSONBackend
+    if solc_version in Spec('>=0.4.11'):
```

Comment:
Just to be sure that you know we can't immediately drop support for `solc==0.4.8`.  Need to raise a `DeprecationWarning` for at least a full minor release across about a 1 month timeline. befor fully dropping it.

Also, see the `0.4.14` release from yesterday.  Security release meaning that we should probably have a more nuanced strategy that raises warnings for anything `>=0.4.11,<0.4.14` to tell people to upgrade because of security issues.